spark/v3.1/spark/src/test/java/com/netease/arctic/spark/TestKeyedTableDML.java

```diff
@@ -108,6 +108,7 @@ public void testSelectChangeFiles() {
         newRecord(keyedTable, 4, "ddd", quickDateWithZone(4)),
         newRecord(keyedTable, 5, "eee", quickDateWithZone(4))
     ));
+    writeBase(identifier,  baseFiles);
```

Comment:
Add more test data.  The change table needs both insert data and delete data.  delete data must contain rows in the base table.